app/helpers/attendings_helper.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module AttendingsHelper
+  def find_by_attendee(attendee)
+    @event.attendees.find_by(id: attendee.id)
+  end
+
+  def find_by_attending(current_user)
+    @event.attendings.find_by(attendee_id: current_user.id)
+  end
```

Comment:
There ought not to be application logic `find_by` methods etc in helpers.
Leave this to controllers. 
You ought to rethink some of your helper methods.
See Sessions helper file for better explanation and checkout the following articles:
Read [this article](http://codefol.io/posts/Where-Do-I-Put-My-Code/) and [this article](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers) for a clearer explanation. 🙂